org.eclipse.jdt.ls.tests/pom.xml

```diff
@@ -108,7 +108,7 @@
 								<artifactItem>
 									<groupId>org.projectlombok</groupId>
 									<artifactId>lombok</artifactId>
-									<version>1.18.32</version>
+									<version>1.18.36</version>
```

Comment:
Note to self, when I do things like https://github.com/redhat-developer/vscode-java/pull/3939, I should probably bump this as well. Thanks :+1: